.github/workflows/scala.yml

```
@@ -0,0 +1,21 @@
+name: Scala CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [1.8, 11, 14, 15]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK ${{matrix.java}}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{matrix.java}}
```

I prefer to put spaces around the variables in GH Actions config, but that's just my personal preference:

```suggestion
    - name: Set up JDK ${{ matrix.java }}
      uses: actions/setup-java@v1
      with:
        java-version: ${{ matrix.java }}
```

---

Done